media/transcoder/transcoder_test.go

```diff
@@ -630,67 +632,68 @@ func testJobWithSetNumberImagesSpritesheet(t *testing.T, projectNumber string, i
 	})
 }
 
+// TODO: Uncomment after fixing https://github.com/GoogleCloudPlatform/golang-samples/issues/2432 (t.Skip not used due to size of test)
 // testJobWithPeriodicImagesSpritesheet tests major operations on a job created from an ad-hoc configuration that
 // generates two spritesheets. It will wait until the job successfully completes as part of the test.
-func testJobWithPeriodicImagesSpritesheet(t *testing.T, projectNumber string, inputURI string, outputURIForPeriodicSpritesheet string) {
```

Comment:
No need to comment this func out, just don't call it for now.

Reply:
I got `declared but not used` until I commented out the func.

Reply:
I'm sorry, you're right. The error was for a variable. Will update.

Reply:
Fixed, thanks.